zebrad/tests/acceptance.rs

```diff
@@ -972,11 +981,21 @@ fn sync_until(
 
     // Use a persistent state, so we can handle large syncs
     let mut config = persistent_test_config()?;
-    // TODO: add convenience methods?
     config.network.network = network;
     config.state.debug_stop_at_height = Some(height.0);
     config.mempool.debug_enable_at_height = mempool_behavior.enable_at_height();
 
+    // Download the parameters at launch, if we're going to need them later.
+    if height > network.mandatory_checkpoint_height() {
+        config.consensus.debug_skip_parameter_preload = false;
+    }
+
+    // Use the default lookahead limit if we're syncing lots of blocks.
+    // (Most tests use a smaller limit to minimise redundant block downloads.)
+    if height > MIN_HEIGHT_FOR_DEFAULT_LOOKAHEAD {
```

Comment:
Minor: should this be `>=` instead?

```suggestion
    if height >= MIN_HEIGHT_FOR_DEFAULT_LOOKAHEAD {
```

Reply:
If we're syncing exactly to the default lookahead limit (2000 blocks), then the minimum lookahead of 800 blocks is fine.

If we changed to `>= 2000` blocks, then as soon as we committed the first checkpoint, we'd download ~400 unnecessary blocks.

Maybe this should be `2 *MIN_HEIGHT_FOR_DEFAULT_LOOKAHEAD`, but it really doesn't matter either way.